dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/FavTaskController.java

```diff
@@ -84,8 +84,8 @@ public Result listTaskType(@ApiIgnore @RequestAttribute(value = Constants.SESSIO
     @ApiException(DELETE_TASK_TYPE_ERROR)
     @AccessLogAnnotation(ignoreRequestArgs = "loginUser")
     public Result deleteFavTask(@ApiIgnore @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
-                                @PathVariable("taskName") String taskName) {
-        boolean b = favTaskService.deleteFavTask(loginUser, taskName);
+                                @PathVariable("taskName") String taskType) {
```

Comment:
```suggestion
                                @PathVariable("taskType") String taskType) {
```

Reply:
emmm..i'm trying to use @TableField avoid this, if change need to change frontend. what's your opinion.

Reply:
It's Ok to change the front end.

Reply:
I think it is ok too

Reply:
done